config/application.yml.default

```diff
@@ -576,6 +576,7 @@ test:
   verify_gpo_key_max_attempts: 2
   verify_personal_key_attempt_window_in_minutes: 3
   verify_personal_key_max_attempts: 2
+  usps_ipp_root_url: 'http://localhost:1000'
```

Comment:
This is necessary because of [this change](https://github.com/bblimke/webmock/compare/v3.15.2...v3.16.2#diff-99d09d8e9c633be1c28a1f63bfb3277d3b7a6cf62f382cb2e0e8500b6732fb8aR137)

Without it, it will fail to parse the URL and raise an error like the following:

```
expected UspsInPersonProofing::Exception::RequestEnrollException, got #<RSpec::Mocks::MockExpectationError:"#<ActiveSupport::Logger:0x0000000125656e60 @level=0, @progname=..."Exception raised while enrolling user: Absolute URI missing hierarchical segment: 'http://:80'\")"> with backtrace:
```